src/components/Feed/Feed.css

```diff
@@ -13,10 +13,22 @@ p {
 }
 
 header {
+    position: fixed;
```

Comment:
It shouldn't be too hard to segment this to solve the issue regarding moving this into its own component since all the styles are in this header ruleset.